src/apps/frontend/components/flex/flex.component.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import clsx from 'clsx';
+import styles from './flex.styles';
+
+interface FlexProps {
+  children: React.ReactNode;
+  direction?: string;
+  justifyContent?: string;
+  alignItems?: string;
+  flexWrap?: string;
+  gap?: number;
+}
+
+const Flex: React.FC<FlexProps> = ({
```

Comment:
`const Flex: React.FC<PropsWithChildren<FlexProps>> = ({` and remove `children` from `FlexProps`